components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/ResourceTypeApi.java

```diff
@@ -0,0 +1,148 @@
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 409, message = "Conflict"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceTypePut(@ApiParam(value = "This represents the resource type to be added.", required = true) ResourceTypeAddDTO type) {
+
+        return delegate.resourceTypePut(type);
+    }
+
+    @DELETE
+    @Path("/{resource-type-name}")
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Revoke resource type.\n", notes = "This API is used to delete an existing resource type.\n", response = void.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 409, message = "Conflict"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceTypeResourceTypeNameDelete(@ApiParam(value = "This represents the resource type to be revoked.", required = true) @PathParam("resource-type-name") String resourceTypeName) {
+
+        return delegate.resourceTypeResourceTypeNameDelete(resourceTypeName);
+    }
+
+    @GET
+    @Path("/{resource-type-name}")
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Get the resource type.\n", notes = "This API is used to get an existing resource type.\n", response = ResourceTypeDTO.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 409, message = "Conflict"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceTypeResourceTypeNameGet(@ApiParam(value = "This represents the resource type to be retrieved.", required = true) @PathParam("resource-type-name") String resourceTypeName) {
+
+        return delegate.resourceTypeResourceTypeNameGet(resourceTypeName);
+    }
+}
+
```

Comment:
remove unnecessary new line

Reply:
Intentionally left due to a generated class.